backend/tests/test_formatters.py

```diff
@@ -1,47 +1,28 @@
 import pytest
 from defusedxml import ElementTree
 
-from backend.digesters.raw import DigesterRaw
+from backend.collections.kiwix_public import KiwixPublicCollection
+from backend.formatters.kiwixlibraryxml import KiwixLibraryXml
 from backend.models import Book
 
 
 @pytest.mark.asyncio
-async def test_raw_endpoint(client, title_with_data):
-    response = await client.get("/v1/digesters/raw")
-    assert response.status_code == 200
-    assert response.headers.get("Content-Type") == "application/xml"
+async def test_kiwix_xml_formatter(book_with_metadata):
+    xml_bytes = await KiwixLibraryXml.generate(collection=KiwixPublicCollection())
+    tree = ElementTree.fromstring(xml_bytes)
 
-    tree = ElementTree.fromstring(response.content)
-    assert "version" in tree.attrib
+    books = tree.findall("book")
 
-    books = list(tree)
+    assert "version" in tree.attrib
     assert len(books) == await Book.objects.count()
+
     for child in books:
         book = await Book.objects.get(id=child.get("id"))
 
         assert book.article_count == int(child.get("articleCount"))
         assert book.media_count == int(child.get("mediaCount"))
         assert book.url == child.get("url")
 
```

Comment:
Can we test that the most common metadata are present and not null ? `title`, `description`, `favicon`, `language`, `name`, `tags`, `date`, `faviconMimeType` ?